### PR TITLE
feat: add Burner Wallet Recovery flow

### PR DESCRIPTION
**Pull request by helciofranco**
- Closes https://github.com/FuelLabs/fuel-connectors/issues/172
- Closes https://github.com/FuelLabs/fuel-connectors/issues/177

---

Add `Burner Wallet` recovery feature.
> A dialog will now prompt the user to recover their previous Burner Wallet session.

Also, fix connection status issue when refreshing the page (it was losing the connection status).

| 📷 Demo |
| --- |
| <video src="https://github.com/user-attachments/assets/ecb21e96-f7bb-4446-b697-03afa2642b8f" /> |

### CODE REVIEW

**Review by arthurgeron — DISMISSED** [review]

**Comment by arthurgeron** [comment]
LGTM 🚀

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
can we generate a npm `pr-version` and test it out on fuel bridge?

**Comment by helciofranco** [comment]
@LuizAsFight 

It's looking good! Check it out
https://fuel-explorer-czzwfrfmr-fuel-labs.vercel.app/bridge?from=eth&to=fuel

| 📷 Demo |
| --- |
| <video src="https://github.com/user-attachments/assets/c81f1367-66b9-4b40-a3fb-43eab8d68d19" /> |

File: .changeset/popular-seahorses-kiss.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+"@fuel-connectors/burner-wallet-connector": minor
+---
+
+Add `Burner Wallet` recovery feature.
+> A dialog will now prompt the user to recover their previous Burner Wallet session.
+
+Also, fix connection status issue when refreshing the page (it was losing the connection status).
```
Comment (LuizAsFight) [inline .changeset/popular-seahorses-kiss.md:8]: can we split this in 2 changesets ?

1- Add Dialog that will prompt the user to recover his previous Burner Wallet session
2- Fix connection being lost when refreshing the page

**Review by LuizAsFight — CHANGES_REQUESTED** [review]

File: .changeset/popular-seahorses-kiss.md (hunk 1 above)
Comment (helciofranco) [reply .changeset/popular-seahorses-kiss.md]: https://github.com/FuelLabs/fuel-connectors/pull/178/commits/522edde141effa771ea9712fab637d3e28de6b62

**Review by LuizAsFight — APPROVED** [review]

**Pull request merged**